src/main.js

```diff
@@ -67,36 +169,39 @@ async function submitHandler() {
             throw new Error('Failed to get first step');
         }
         
-        handleFirstStep(firstStep);
+        await handleNextStep(firstStep);
 
         const secondStep = await FRAuth.next(firstStep);
         console.log('secondStep', secondStep);
         if (!secondStep) {
             throw new Error('Failed to get second step');
         }
         
-        const accessToken = await handleSecondStep(secondStep);
-        if (!accessToken) {
-            throw new Error('Failed to get access token');
-        }
-
-        // Display user info
-        const user = await UserManager.getCurrentUser();
-        if (!user) {
-            throw new Error('Failed to get user');
-        }
-        userInfoElem.innerHTML = JSON.stringify(user, null, 2);
-        userPanel.style.display = 'block';
-        successElem.style.display = 'block';
-        formElem.style.display = 'none';
+        await handleNextStep(secondStep);
     } catch (err) {
         console.error('Failed to submit: ', err);
         errorElem.style.display = 'block';
     }
 }
 
-document.getElementById('login-form').addEventListener('submit', submitHandler);
+function submitChoiceHandler(step) {
+    const choiceCallback = step.getCallbackOfType('ChoiceCallback');
+    const choices = document.querySelectorAll('input[name="choice"]');
+    let selectedChoice;
+    choices.forEach(choice => {
+        if (choice.checked) {
+            selectedChoice = choice.value;
+            console.log('selected choice', selectedChoice);
+        }
+    });
+    choiceCallback.setChoiceValue(selectedChoice);
+    nextSteps(step);
+}
+
+loginElem.addEventListener('submit', submitLoginHandler);
```

Comment:
One downside to this multiple forms and submission event handlers is you can have the inadvertent consequence a memory leak from attaching event listeners and not detaching them after their use. That's why having a single form element is often beneficial as you attach the single event listener, and leave it alone.